runtime/frequency/src/xcm/teleporter.rs

```diff
@@ -0,0 +1,27 @@
+use frame_support::parameter_types;
+use staging_xcm::latest::prelude::*;
+use staging_xcm_builder::Case;
+
+/// The location of AssetHub on the relay network.
+pub const ASSET_HUB_ID: u32 = 1000;
```

Comment:
question: is this ID the same for assethub on westend and mainnet?

Reply:
I believe so. We have a ticket to run on Paseo and then on mainnet, so some of these differences will be addressed and tested at that stage. For now, the focus is on Westend.